paper/paper.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Blobmodel: A Python package for generating superpositions of pulses in one and two dimensions'
+tags:
+  - Python
+  - pulses
+  - imaging data 
+  - tokamaks
+  - turbulence
+authors:
+  - name: Juan M. Losada
+    orcid: 0000-0003-2054-1384
+    equal-contrib: true
+    affiliation: 1
+  - name: Gregor Decristoforo
+    orcid: 0000-0002-7616-0946
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 1
+affiliations:
+ - name: UiT, The Arctic University of Norway
+   index: 1
+date: 11 February 2025
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`blobmodel` is a Python library for generating synthetic data that mimics the behavior
+of moving pulses in a turbulent environment. It creates controlled datasets where the
+true motion of each pulse is known, allowing researchers gain further understanding on
+the statistical outputs of such systems as well as to test and improve analysis 
+and tracking algorithms. While originally developed for studying
+turbulence in fusion experiments, `blobmodel` can be applied to any field where
+turbulence leads to the generation of structures propagating in one or two dimensions.
+The software is open source, easy to use, and designed to support reproducible research.
+
+# Statement of need
+
+Understanding and analyzing the motion of structures in turbulent systems is crucial
+in many areas of research, including plasma physics [@dippolito_convective_2011],
+fluid dynamics [@fiedler_coherent_1988], and atmospheric science [@nosov_coherent_2009]. 
+
+More widely, the study of the statistical characteristics resulting from the superposition
+of uncorrelated propagating pulses is of importance in fields such as astrophysical plasmas [@veltri_mhd_1999], 
+detection rates of interplanetary dust [@kociscak_modeling_2023], 
+$1/f$ noise in self-organized critical systems [@bak_self_1988], 
+shot noise in electronics [@lowen_power_1990]
+
+In experimental studies, imaging diagnostics are often used to capture the 
+evolution of these structures [@zweben_invited_2017], but extracting reliable velocity information from such 
+data remains challenging [@offeddu_analysis_2023]. Many existing analysis methods rely on assumptions about 
+the underlying dynamics and must be tested against known reference data to ensure
+accuracy.
+
+Several stochastic models have been developed describing a superposition of uncorrelated structures arriving in time
+in zero spatial dimensions [@garcia_stochastic_2016]; or propagating in one [@losada_stochastic_2023]
```

Comment:
It is not clear to me what is meant by "arriving in time in zero spatial dimensions"